massgen/mcp_tools/hooks.py

```
@@ -879,7 +1078,12 @@ async def call_tool(
 
             except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)